src/pages/Travel/DomainSelectorPage.tsx

```diff
@@ -32,8 +30,8 @@ function DomainSelectorPage({route}: DomainSelectorPageProps) {
     const styles = useThemeStyles();
     const {translate} = useLocalize();
 
-    const [activePolicyID] = useOnyx(ONYXKEYS.NVP_ACTIVE_POLICY_ID, {canBeMissing: true});
-    const policy = usePolicy(activePolicyID);
+    const {policyID} = route.params;
```

Comment:
we need to be more specific than just all params right? Or am I missing something?

Reply:
Nevermind, I was reading this wrong